syntax = "proto3";

package example;

// 定义一个服务
service EchoService {
    rpc Echo(EchoRequest) returns (EchoResponse);
}
option cc_generic_services = true;
// 请求消息
message EchoRequest {
    string message = 1;
}

// 响应消息
message EchoResponse {
    string message = 1;
}
